docs/P--Gutenberg Blocks and Components/README.md

```diff
@@ -0,0 +1,13 @@
+# Gutenberg Blocks and Components
+
+### Blocks
+The following is a list of Event Espresso Gutenberg Blocks that can be utilized in your site's content:
+
+ * [Event Attendees List](blocks/event-attendees.md) 
+ 
+### Components
+If you are building your own Gutenberg Block to use within Event Espresso, then the following is a list of additional React components can be utilized:
+ 
+ 
+ * [QueryLimit](components/query/limit.md) - A React component for adding a WordPress Range Control input for setting a query's limit clause
+
```

Comment:
Gutenberg is a code name and I'm not sure we should use it in our documentation.  In the case of the folder structure it means that if we remove the name down the road it will break any direct bookmarks people have made. 

So I have a couple suggestions:

- `P--Blocks-and-Components`  (please use dashes because they are easier to work with for urls).  
- or nest it in the existing `AA-Javascript-in-EE` folder so it would be `AA-Javascript-in-EE/Blocks-and-Components/`  I prefer this option because this stuff is _mostly_ js related.  However, I'm not going to be sticky on this point because I also see the value of it being a top-level folder.

Reply:
will rename and move